src/test/java/com/epam/tc/hw3/ex1/IndexPageTest.java

```diff
@@ -0,0 +1,144 @@
+package com.epam.tc.hw3.ex1;
+
+import com.epam.tc.hw3.BasePageTest;
+import com.epam.tc.hw3.PropertyReader;
+import com.epam.tc.hw3.components.LoginComponent;
+import com.epam.tc.hw3.fluent.IndexFluentPage;
+import com.epam.tc.hw3.voids.IndexPage;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class IndexPageTest extends BasePageTest {
+
+    @Test
+    public void testIndexPageWithVoidPages() {
+        SoftAssertions softAssertions = new SoftAssertions();
```

Comment:
Please move all the same steps in both tests and classes (for DifferentElementsPageTest the same steps) in base class. Now it's code duplicate.

Reply:
Common steps were moved to utils classes